will/plugins/productivity/remind.py

```diff
@@ -15,7 +15,6 @@ def remind_me_at(self, message, reminder_text=None, remind_time=None, to_string=
             formatted_to_string = ""
         formatted_reminder_text = "%(mention_handle)s, you asked me to remind you%(to_string)s %(reminder_text)s" % {
             "mention_handle": message.sender.mention_handle,
-            "from_handle": message.sender.handle,
```

Comment:
Was this an intentional change?

Reply:
Yes flake8 complained we had from_handle mapped but didn’t use it. I tested the functionality and it still works fine without that line.

Reply:
I actually rewrote it using f’ strings but then though I didn’t need to make as big a change so I just removed that line.

Reply:
Got it, totally fair.